backend/src/main/kotlin/org/loculus/backend/service/submission/CleanUpAuxTableTask.kt

```diff
@@ -0,0 +1,40 @@
+package org.loculus.backend.service.maintenance
+
+import kotlinx.datetime.DateTimeUnit
+import kotlinx.datetime.minus
+import kotlinx.datetime.toLocalDateTime
+import org.loculus.backend.log.AuditLogger
+import org.loculus.backend.service.submission.UploadDatabaseService
+import org.loculus.backend.utils.DateProvider
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Component
+
+private val log = mu.KotlinLogging.logger {}
+
+@Component
+class CleanUpAuxTableTask(
+    private val uploadDatabaseService: UploadDatabaseService,
+    private val dateProvider: DateProvider,
+    private val auditLogger: AuditLogger,
+) {
+
+    /**
+     * Runs every hour and deletes auxTable entries older than 2 hours.
+     */
+    @Scheduled(fixedDelay = 1, timeUnit = java.util.concurrent.TimeUnit.HOURS)
+    fun task() {
+        val hourCutoff = 2L
```

Comment:
```suggestion
        val hourCutoff = 24L
```

While 2 hours is probably fine I can imagine some future situation where 2 hours might be a plausible time and this would be a really annoying cause-of-bug to have to chase down - hopefully 24 hours will never be a problem

Reply:
I suspect we have timeouts so that atm this probably wouldn't work anyway which is why I said "future", but the idea of the aux tables is to stream into the DB from the upload (uncertain exactly to what extent that is the case atm). It is fairly plausible for someone on a slow connection (upload is typically slower than download) to be streaming their upload over a couple of hours.